.gitignore

```diff
@@ -23,5 +24,10 @@ coverage
 build/Release
 
 # Dependency directory
-# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+
```

Comment:
Are you sure you're working on the `.gitignore` file and not the old `gitignore` file? The doc has been saved, staged, and committed, right?